pkg/nack/responder_interceptor.go

```diff
@@ -246,8 +246,10 @@ func (n *ResponderInterceptor) resendPackets(nack *rtcp.TransportLayerNack, last
 				} else {
 					packetsSentWithoutDelay++
 					if n.retransmittedPacketsCount != nil && n.retransmittedPacketsBytes != nil {
+						payload := p.Payload()
 						*n.retransmittedPacketsCount++
-						*n.retransmittedPacketsBytes += uint64(len(payload))
+						*n.retransmittedPacketsBytes += uint64(len(payload)+p.Header().MarshalSize()) + 8 // +8 for underlying UDP header size
```

Comment:
Note that the packet here is not rtp.Packet, but retainablePacket type which doesn't have MarshalSize, so calculating with header + payload..